assets/cluster-version-operator/cluster-version-operator-deployment.yaml

```diff
@@ -54,7 +54,7 @@ spec:
       containers:
         - name: cluster-version-operator
           image: {{ .ReleaseImage }}
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
Please search for `imagePullPolicy: Always` and you will find more parts to change.

Reply:
I've searched the entire repo for more images with ```imagePullPolicy: Always```, but I can't seem to find any. My fork is updated, but I'll check again. I'm sure there will be more to change.

Reply:
See https://github.com/openshift/ibm-roks-toolkit/blob/master/assets/roks-metrics/roks-metrics-deployment.yaml#L38 and https://github.com/openshift/ibm-roks-toolkit/blob/master/assets/roks-metrics/roks-metrics-push-gateway-deployment.yaml#L39

Reply:
Thank you Richard, I realized that I changed those files but didn't save them, which is why they didn't show up when I searched for them.